irc/lchat/Portfile

```
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                lchat
+version             1.0
+categories          irc
+maintainers         @wwalexander openmaintainer
+license             MIT
+description         line oriented front end for ii-like chat programs
+long_description    lchat (line chat) is a line oriented front end for ii-like chat programs. It handles the input from keyboard and output file in parallel. Thus, you are able to type messages while new chat lines are arriving. Its main focus is on usability and simplicity.
+set domain          suckless.org
+homepage            https://tools.${domain}/${name}/
+master_sites        https://dl.${domain}/tools/
+checksums           rmd160  ddbbb2027ccc295ed3f6fd9d04fdc6619a8d21f7 \
+                    sha256  89247b5c8e853bbfc2f97909b1926fadf44d637543767e77e9e42e72242f375f \
+                    size    24455
+patchfiles          config.diff
+depends_lib         port:libgrapheme
+depends_run         bin:/usr/bin/tail:coreutils bin:/usr/bin/grep:grep
```

I'm not sure that we're OK with directly referencing the built-in macOS utilities in `/usr/bin` as dependencies for ports in MacPorts.

This might be better written as simply:
```suggestion
depends_run         bin:tail:coreutils bin:grep:grep
```

CC @ryandesign for thoughts.

---

Makes sense, I was wondering if this was possible; I should have tested this. Thanks for the tip!

---

@wwalexander Please push an updated version of the portfile.